src/seedu/addressbook/data/person/Address.java

```diff
@@ -25,8 +31,17 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
         if (!isValidAddress(address)) {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
+        splitAddress(address);
         this.value = address;
     }
+    
+    public void splitAddress(String address){
+    	String[] addresses = address.split(ADDRESS_SPLITER);
+    	block = new Block(addresses[0]);
```

Comment:
Indentation problem here?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.